Libplanet/Blocks/Block.cs

```diff
@@ -126,9 +135,11 @@ public Block(Bencodex.Types.Dictionary dict)
         {
         }
 
+        // FIXME: Should this be
```

Comment:
It seems does not write the word to follow.

Reply:
Completed the sentence.